doc/source/starting.rst

```diff
@@ -519,25 +574,40 @@ As with OpenMC, the set logic operations have been mapped to python's bit logic
 * ``~``, the not operator, represents a set complement.
 
 .. note::
+
    When you combine two half-spaces with a logical operator you create a new half-space.
    In this case the concept of a side becomes much more about "in" and "out".
 
 .. note::
+
    Half-spaces need not be contiguous.
 
 Order of precedence and grouping is automatically handled by python so you can easily write complicated geometry in one-line.
 
-.. code-block:: python
-
+.. testcode::
+
+   # build blank surfaces 
+   bottom_plane = montepy.surfaces.axis_plane.AxisPlane()
+   top_plane = montepy.surfaces.axis_plane.AxisPlane()
+   fuel_cylinder = montepy.surfaces.cylinder_on_axis.CylinderOnAxis()
+   clad_cylinder = montepy.surfaces.cylinder_on_axis.CylinderOnAxis()
+   clad_od = montepy.surfaces.cylinder_on_axis.CylinderOnAxis()
+   other_fuel = montepy.surfaces.cylinder_on_axis.CylinderOnAxis()
```

Comment:
So, I get that these objects are instantiated without surface coefficients, but it makes for a weird example not using any.

Reply:
Should I add coefficients?

Reply:
yes.